### ISSUE TITLE
Run Store + bundles in a worker?

### ISSUE THREAD

**Issue by developit** [HenrikJoreteg/redux-bundler#2]
The use of strings for selectors and actions in `connect()` would make for a really nicely optimized worker boundary, better than Stockroom has been able to pull off.
Any thoughts on how one might push redux and bundled actions into a worker?

**Comment by HenrikJoreteg** [comment]
I'll work on putting together an example app when I get a chance. This is actually how I started building the app I just shipped, but ended up backing it out because I was using Firebase and running it in a worker, while it seemed to work, isn't officially supported and plus, it just felt kinda risky to do something more experimental like this in an app that's handling real financial transactions. So I backed it out, but... it's certainly doable.
